paseos/__init__.py

```diff
@@ -27,4 +27,4 @@ def init_sim(local_actor: BaseActor):
     return sim
 
 
-__all__ = ["ActorBuilder", "GroundstationActor", "SpacecraftActor"]
```

Comment:
I think you need to also import it, at least
`from paseos import SpaceAnimation`
does not work

Reply:
I added a visualization_example folder where one can run a minimal example.

Reply:
I think you misunderstood, the code still only works if you are inside the paseos folder, because you are missing `from paseos import SpaceAnimation` in this file

Reply:
![image](https://user-images.githubusercontent.com/2656685/198342172-76d865e1-21ea-4e73-8b27-640bd0d4ff67.png)

top of this file needs to look like this or you get an import error when trying to import it